Public/Src/Utilities/Configuration/ISandboxConfiguration.cs

```diff
@@ -243,5 +245,15 @@ public interface ISandboxConfiguration
         /// Thus, this enforcement is made opt-in.
         /// </remarks>
         bool EnsureTempDirectoriesExistenceBeforePipExecution { get; }
+
+        /// <summary> 
+        /// Paths which should be untracked for all processes 

```

Comment:
>Paths [](start = 12, length = 5)

Directory paths.

Reply:
It is not only for Directories, this list can also contain file paths.

Reply:
But it has "Scopes" as suffix.

---
In reply to: [292620158](https://github.com/microsoft/BuildXL/pull/442#discussion_r292620158) [](ancestors = 292620158)

Reply:
Clarifid in remarks